consistency/tool/clustering/clustering_test.json

```diff
@@ -44,6 +44,45 @@
          "words": ["Abandon", "Collaborative","Duplication","Environments","Stragified"], 
          "labels": [-1,-1,-1,-1,-1]
       }
-  }
+  }, 
+  "form-ids": {
+    "noise-words": { 
+      "words": ["Abandon", "Collaborative", "Duplication","Environments","Stragified"], 
+      "labels": [-1,-1, -1, -1, -1],
+      "clustered-words": {"NOISE_WORDS": ["Abandon", "Collaborative", "Duplication","Environments","Stragified"]},
+      "parsable-list": [{"id": "NOISE_WORDS",
+                         "kind": "WordGroup",
+                         "word_frequency": {
+                                            "Abandon":1,
+                                            "Collaborative":1,
+                                             "Duplication":1,
+                                             "Environments":1,
+                                             "Stragified":1
+                                          }
+                         }]
+   }, 
+   "duplicate-words": { 
+      "words": ["a", "a", "b", "b", "c"], 
```

Comment:
Since we have a condition of `len(word)>2` for a word to be in the list of valid_words,  can we update the examples here, just to make sure we have test cases with valid words